ethereum/datastructures/src/test/java/tech/pegasys/artemis/datastructures/util/CommitteeUtilTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.datastructures.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.stream.IntStream;
+import org.apache.tuweni.bytes.Bytes32;
+import org.junit.jupiter.api.Test;
+
+public class CommitteeUtilTest {
+
+  @Test
+  void testListShuffleAndShuffledIndexCompatibility() {
+    Bytes32 seed = Bytes32.ZERO;
```

Comment:
Given how comprehensive the reference tests are for shuffling, we can probably delete the original implementation and ditch this test.  *Maybe* hard code one expected shuffle result here just as a quick sanity check as part of unit tests and leave the details to the reference tests.

Reply:
The original `compute_shuffled_index` is still actual for computing proposer index. So me should leave it and may leave this simple test

Reply:
ah ok, I thought we'd be able to delete it, so yes testing that they give the same results is definitely the best approach then.  Guess that explains how you got away with only doing backwards search and not forwards...